src/fu-release.c

```diff
@@ -942,6 +942,7 @@ fu_release_load(FuRelease *self,
 
 	/* check requirements for device */
 	if (self->device != NULL && self->request != NULL &&
+	    (install_flags & FWUPD_INSTALL_FLAG_NONE) != 0 &&
```

Comment:
I think I would feel better about this to have a new install flag for ignore requirements explicitly set by the get_details flow.  There may be multiple callers into this function that might want to evaluate requirements even if INSTALL_FLAG_NONE wasn't set.

Reply:
I've added `FWUPD_INSTALL_FLAG_NO_REQUIREMENTS` install flag and introduced it from 1.8.2 onward.